evo-delivery-app/evo-delivery-backend/src/clients/utils/EaHttpConverter.ts

```diff
@@ -1,80 +1,96 @@
 import {
-    Driver,
-    EaHttpRequestDriver,
-    Order,
-    EaHttpRequestOrder,
-    DistanceMatrix,
-    EaHttpRequestDistances,
-    EaEvaluateConfig,
-    Depot, EaEvaluateHttpRequestBody,
-} from '../../types';
-import {convertObjCamelToSnakeCase} from '../../utils';
+  Driver,
+  EaHttpRequestDriver,
+  Order,
+  EaHttpRequestOrder,
+  DistanceMatrix,
+  EaHttpRequestDistances,
+  EaEvaluateConfig,
+  Depot,
+  EaEvaluateHttpRequestBody,
+} from "../../types";
+import { convertObjCamelToSnakeCase } from "../../utils";
 
 export interface EaHttpConverter {
-    toEaHttpRequestBody(drivers, orders, depot, distanceMatrix, config): EaEvaluateHttpRequestBody;
+  toEaHttpRequestBody(
+    drivers,
+    orders,
+    depot,
+    distanceMatrix,
+    config,
+    runId?
+  ): EaEvaluateHttpRequestBody;
 
-    convertDepot(depot: Depot): string;
+  convertDepot(depot: Depot): string;
 
-    convertDriver(driver: Driver): EaHttpRequestDriver;
+  convertDriver(driver: Driver): EaHttpRequestDriver;
 
-    convertOrder(order: Order): EaHttpRequestOrder;
+  convertOrder(order: Order): EaHttpRequestOrder;
 
-    convertDistances(distanceMatrix: DistanceMatrix): EaHttpRequestDistances;
+  convertDistances(distanceMatrix: DistanceMatrix): EaHttpRequestDistances;
 
-    convertConfig(config: EaEvaluateConfig)
+  convertConfig(config: EaEvaluateConfig);
 }
 
 export class EaHttpConverterImpl implements EaHttpConverter {
-    toEaHttpRequestBody(_drivers, _orders, _depot, _distanceMatrix, _config): EaEvaluateHttpRequestBody {
-        const drivers: EaHttpRequestDriver[] = _drivers.map(this.convertDriver);
-        const orders: EaHttpRequestOrder[] = _orders.map(this.convertOrder);
-        const distances: EaHttpRequestDistances = this.convertDistances(_distanceMatrix);
-        const depotId = this.convertDepot(_depot);
-        const config = this.convertConfig(_config);
+  toEaHttpRequestBody(
+    _drivers,
+    _orders,
+    _depot,
+    _distanceMatrix,
+    _config,
+    _runId?
+  ): EaEvaluateHttpRequestBody {
+    const drivers: EaHttpRequestDriver[] = _drivers.map(this.convertDriver);
+    const orders: EaHttpRequestOrder[] = _orders.map(this.convertOrder);
+    const distances: EaHttpRequestDistances =
+      this.convertDistances(_distanceMatrix);
+    const depotId = this.convertDepot(_depot);
+    const config = this.convertConfig(_config);
 
-        return {
-            data: {drivers, orders, distances, root_id: depotId},
-            config,
-        } as EaEvaluateHttpRequestBody;
+    return {
+      run_id: _runId,
```

Comment:
here

Reply:
its can't be changed because the ea_server expect the request body be with specific name